sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -301,7 +301,7 @@ CREATE TABLE `onsite_documents` (
 CREATE TABLE `onsite_mail` (
   `id` bigint(20) NOT NULL AUTO_INCREMENT,
   `date` datetime DEFAULT NULL,
-  `owner` bigint(20) DEFAULT NULL,
+  `owner` varchar(128) DEFAULT NULL,
```

Comment:
Since this has been in the codebase awhile, will need to add change for this at the end of this script instead:
``````
#IfNotColumnType onsite_mail owner varchar(128)
ALTER TABLE `onsite_mail` CHANGE `owner` `owner` varchar(128);
#Endif